Embrace Unity iOS Interface/Sources/EmbraceManager.swift

```diff
@@ -46,7 +46,11 @@ public class EmbraceManager: NSObject {
             
             return true
         } catch let e {
-            os_log("Error starting Native Embrace SDK \(e.localizedDescription)")
```

Comment:
curious if these changes were prompted by a customer issue? We do the same `os_log` call in the React Native SDK, should we have a similar check there?

Reply:
I've changed the SDK to have iOS 13 as the `MINIMUM_DEPLOYMENT_TARGET` ; that means that `os_log` APIs (at least the ones that we were using here) are only available if the OS is `14.0` or above.

Reply:
Did you change that in the xcodegen `project.yml` file or the original `.xcodeproj`?

Reply:
The original `.xcodeproj`: (aka `Embrace Unity iOS Interface/Embrace Unity iOS Interface.xcodeproj/project.pbxproj`)

Reply:
got it, ya we only specify 13 as our minimum in React Native, ok we should make this change there as well